genkit-tools/config/nextjs.genkit.ts.template

```diff
@@ -38,3 +34,8 @@ const jokeFlow = defineFlow(
     return llmResponse.text();
   }
 );
+
+export async function callMenuQAFlow() {
```

Comment:
I think we can leave this off.

Reply:
Pavel originally included this because nextjs server code can only export async functions. We can't export the flow for nextjs to use so we generate a sample async function that they should call as a server action.